### DISCUSSION TITLE
Aula 'Tela de Produtos' interrompida

### DISCUSSION THREAD

**Discussion by Robson-Teixeira** [balta-io/7187#81] [Perguntas e Respostas] [answered]
Boa noite!

A aula '[Tela de Produtos](https://balta.io/player/assistir/5d493a0c-7c21-3e60-b8eb-f5ef00000000/8f4710c6-baf7-4b98-b028-000071870113)' está parando em 1:32. 
Fiz o teste de reproduzi-la em outros dispositivos e em redes diferentes, mas ainda assim não obtive sucesso.

![image](https://user-images.githubusercontent.com/23440285/166404615-ca088bb4-5f42-4a7e-bd27-79a63e729a93.png)

Desde já, obrigado!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @Robson-Teixeira, como vai? Fiz um teste aqui e está normal... deve ter sido alguma instabilidade na sua rede... Pode testar novamente por favor?

**Reply by Robson-Teixeira** [reply]
Bom dia André! Tudo joia, e aí?
Continua com problema... Chega em 1:32 e ele pula pro final. 
Fiz o teste de outras redes e em dispositivos distintos, mas tudo sem sucesso.

**Reply by andrebaltieri** [reply]
Obrigado pelo feedback @Robson-Teixeira, estamos trabalhando nisto!
